src/main/java/com/shanebeestudios/skbee/elements/fishing/expressions/ExprFishingExperience.java

```diff
@@ -0,0 +1,94 @@
+package com.shanebeestudios.skbee.elements.fishing.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.Experience;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerFishEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Fishing - Experience")
+@Description("Get and modify the amount of experience dropped in a fishing event")
```

Comment:
Period at the end of the line
```suggestion
@Description("Get and modify the amount of experience dropped in a fishing event.")
```